### PR TITLE
Explore: parse time range fix

### LINKED ISSUE #28463: "Unexpected error happened" following Loki -> Tempo flow
Screenshot 2020-10-22 at 10 56 46

**What happened**:

Follow the metric->logs->tempo flow, get unexpected error when clicking on a trace id in explore.

**What you expected to happen**:

Showing the trace.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: Grafana v7.4.0-5134pre (a0932f4d2a)
- Data source type & version: 
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by zoltanbedi**
**Which issue(s) this PR fixes**:

Fixes #28463

### CODE REVIEW

File: public/app/core/utils/explore.ts [hunk 1]
```diff
@@ -374,10 +380,11 @@ const parseRawTime = (value: any): TimeFragment | null => {
     return toUtc(value, 'YYYY-MM-DD HH:mm:ss');
   }
 
-  if (!isNaN(value)) {
-    const epoch = parseInt(value, 10);
-    return toUtc(epoch);
-  }
+  // [Zoltan] There shouldn't be a number here
```
Comment (aocenas) [inline public/app/core/utils/explore.ts:383]: I see in this case why this was removed. Probably makes sense, as we type this better if we put number here hopefully it should trigger compile time errors somewhere

Comment (zoltanbedi) [reply public/app/core/utils/explore.ts:383]: yep. so what now? should I remove this? or uncomment and add back any?

Comment (zoltanbedi) [reply public/app/core/utils/explore.ts]: removed it

**Review by aocenas — APPROVED** [review]

**Comment by tomwilkie** [comment]
Thanks Zoltan & Andrej!

**Pull request merged**